src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -29,74 +30,142 @@ export const chartShowValuesPlugin: Plugin = {
     if (!drawData) {
       return;
     }
-    const ctx = chart.ctx;
+    const ctx = chart.ctx as CanvasRenderingContext2D;
```

Comment:
think its not necessary.

Reply:
Not strictly necessary, but helpful when coding to have VSCode suggest the different canvas context's methods

Reply:
didn't realize it was types as nay, my bad  :+1: